styleguide/_sass/_rsweb-font-style.scss

```diff
@@ -1,10 +1,10 @@
 @font-face {
     font-family: 'rsweb';
-    src:    url('../fonts/rsweb.eot?ezv9m0');
-    src:    url('../fonts/rsweb.eot?ezv9m0#iefix') format('embedded-opentype'),
-        url('../fonts/rsweb.ttf?ezv9m0') format('truetype'),
-        url('../fonts/rsweb.woff?ezv9m0') format('woff'),
-        url('../fonts/rsweb.svg?ezv9m0#rsweb') format('svg');
+    src:    url('https://0d0fbe434114a2d40b45-de9ed8c45372075d79906316dada8cf3.ssl.cf1.rackcdn.com/icon/rsweb.eot?ezv9m0');
```

Comment:
Amber didn't have any perspective on the qstrings.  Gutcheck says it was to bust cache.  I removed them and they work just fine (although the #iefix hashtag should stay).  

Re: the protocol, the files are hosted directly from the SSL version of the container, so mixed protocol won't work here.  Especially when you are testing local, which is not https.  Confirmed via visual test that stripping the protocol breaks this.